src/abilities/Golden-Wyrm.js

```diff
@@ -351,24 +366,10 @@ export default (G) => {
 			 * This is a total of 10 hexes assuming the Golden Wyrm is not adjacent to
 			 * the edge of the board.
 			 *
-			 * @returns Refer to HexGrid.getHexMap()
+			 * @returns {Hex[]} Refer to Creature.adjacentHexes()
 			 */
 			_getHexes() {
-				const map = [
-					[1, 1, 1, 1, 0],
-					[1, 0, 0, 0, 1],
-					[1, 1, 1, 1, 0],
-				];
-				const xOffset = this.creature.y % 2 === 0 ? 0 : 1;
-				const hexes = G.grid.getHexMap(
-					this.creature.x - xOffset,
-					this.creature.y - 1,
-					-2,
-					false,
-					map,
-				);
-
-				return hexes;
+				return this.creature.adjacentHexes(1);
```

Comment:
This is the hex selection helper function I was hoping existed!

Reply:
Yeah, well, it was too obvious. Guessing you have to go back to some of your previous work to remove own implementation?

Reply:
Yes, this PR changes Visible Stigmata to use the same helper :)